README.md

```diff
@@ -119,6 +119,19 @@ You can use the `wp_parsely_metadata` filter, which sends three arguments: the a
 
 This filter can go anywhere in your codebase, provided it always gets loaded.
 
+### How can I access the JavaScript hooks on the Parse.ly object?
+
+For some advanced integrations, you need to hook custom JavaScript code to some events on the `window.PARSELY` object. To do so, our plugin supports WordPress JavaScript hooks. For example:
+
+```
+$script = '
+window.wpParselyHooks.addAction("wpParselyOnLoad", "wpParsely", testFunc, 10);
+function testFunc() {
+	console.log("This is a hook");
+}';
+wp_add_inline_script( 'wp-parsely-loader', $script );
+```
```

Comment:
```suggestion
~~~php
$script = '
window.wpParselyHooks.addAction("wpParselyOnLoad", "wpParsely", testFunc, 10);
function testFunc() {
	console.log("This is a hook");
}';
wp_add_inline_script( 'wp-parsely-loader', $script );
~~~
```

README markdown should use tilda instead of backtick fences for code blocks so that it renders correctly on WordPress.org.